README.md

```diff
@@ -52,6 +52,29 @@ The `get_trigger.ts` file is executed as a Deno program and takes one required a
 deno run -q --config=deno.jsonc --allow-read --allow-net https://deno.land/x/deno_slack_hooks/get_trigger.ts --source="./trigger.ts"
 ```
 
+### Generate Trigger Script Usage
+
+The `get_trigger.ts` file is executed as a Deno program and takes one required argument:
+
+| Arguments  | Description                                           |
```

Comment:
Can you please add information about this hook to the [Slack CLI/SDK interface specification](https://corp.quip.com/0gDvAsqoaaYE/CLI-SDK-Interface-Specification-slackjson#temp:C:fOC1991c5aec8994d0db01d26260), please?

Reply:
Done (for 'generate_trigger')! Hadn't realised we had this document